float64 T_p
float64 Alpha_p
float64 T_s
float64 Alpha_s
